weldx/utility.py

```diff
@@ -142,25 +154,52 @@ def to_pandas_time_index(time) -> Union[pd.TimedeltaIndex, pd.DatetimeIndex]:
 
     Returns
     -------
-    Union[pandas.TimedeltaIndex, pandas.DatetimeIndex]:
-        Variable as pandas time index
+    pandas.TimedeltaIndex or pandas.DatetimeIndex
+        Time union of all input objects
```

Comment:
We should definitely mention the capabilities of this function in the developer's documentation. Otherwise, it might get overlooked in the future since the naming doesn't give much of a hint here.

Reply:
Maybe add to time-related dev doc?